apps/factory/src/pages/_app.tsx

```diff
@@ -15,13 +16,31 @@ type AppPropsWithLayout = AppProps & {
   Component: NextPageWithLayout;
 };
 
+export function reportWebVitals({
```

Comment:
This function is used by "nextjs-google-analytics" package 
Docs -- https://www.npmjs.com/package/nextjs-google-analytics